packages/client/src/generation/utils/buildNFTAnnotations.ts

```diff
@@ -11,22 +11,22 @@ import { map } from '../../../../../helpers/blaze/map'
  * Build bundler-like annotations so that Vercel automatically uploads the
  * prisma schema as well as the query engine binaries to the deployments.
  * @param engineType the client engine in use
- * @param platforms the targeted platforms
+ * @param binaryTargets the targeted platforms
```

Comment:
```suggestion
 * @param binaryTargets the targeted binaryTargets
```
Weird but more accurate?